src/main/java/com/mycompany/worldofzuul/Game.java

```diff
@@ -91,15 +90,94 @@ public Room getRoom(String name) {
 
         return null;
     }
-    
-    public void setCurrentPointOfInterest(PointOfInterest poi){
+
+    public void setCurrentPointOfInterest(PointOfInterest poi) {
         this.currentPointOfInterest = poi;
     }
-    public void setCurrentRoom(Room room){
+
+    public void setCurrentRoom(Room room) {
         this.currentRoom = room;
     }
-    
+
     public PointOfInterest getCurrentPointOfInterest() {
         return currentPointOfInterest;
     }
+
+    public ArrayList<String> createEndList() {
+        // arraylist
+        ArrayList<String> picturepaths = new ArrayList<>();
+        for (Room room : rooms) {
+            int count = 0;
+            double maxcount = 0;
+            for (PointOfInterest pointofinterest : room.getPointsOfInterest()) {
+
+                if (pointofinterest.isFixable()) {
+                    maxcount++;
+
+                    if (pointofinterest.isFixed()) {
+                        count++;
+
+                    }
+                }
+
+            }
```

Comment:
Kan man ikke bare lave et loop som tjekker for hvert rum i stedet for alle de if-statements?